app/experimenter/templates/experiments/field_inline_help_link.html

```diff
@@ -0,0 +1,23 @@
+<div class="form-group row">
+  <div class="col-3 text-right">
+    <label class="col-form-label" for="{{ field.id_for_label }}">
+      <strong>{{ field.label }}</strong>
+      {% if not field.field.required %}
+        <span class="text-muted optional-marker">Optional</span>
+      {% endif %}
+    </label>
+    <br>
+    <a target="_blank" rel="noreferrer noopener" href="{{ field.help_text }}">Help</a>
+  </div>
+
+  <div class="col-9">
+    {% include "experiments/field_input_inline.html" with field=field %}
+
+    {% include "experiments/field_errors_inline.html" with field=field %}
+
+    <div id="{{ field.id_for_label }}-help" class="text-muted collapse {% if field.errors %}in{% endif %} pt-2">
+      {{ field.help_text|safe }}
+    </div>
+
+  </div>
+</div>
```

Comment:
I added this template because the existing ones (`form-inline`) did not have the orientation where there is a help button that opens a separate window with a link (as opposed to popping up some help text under the textfield box)..

Reply:
I think this would make a great comment at the top of this template!